app/graphql/mutations/consultation/create.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  module Consultation
+    class Create < Mutations::BaseMutation
+      type Types::Objects::Consultation::Base, null: false
+
+      argument :consultation, Types::Inputs::Consultation::Create, required: true
+
+      def resolve(consultation:)
+        created_consultation = ::Consultation.new consultation.to_h
+        created_consultation.created_by = context[:current_user]
```

Comment:
@mikevic  - You are not using Current.user in this project?

Reply:
Yes, for now, I am just sticking to context since I don't really see the need for Current yet.

Reply:
okay bro.